docs/examples/test/test_examples.py

```diff
@@ -286,27 +286,26 @@ def test_websocket_rpc(self, container_factory, web_config, websocket):
 
 class TestConfig:
 
-    def test_config_value_not_set(self, container_factory, empty_config):
+    def test_config_value_not_set(self, container_factory, rabbit_config):
```

Comment:
The service that it starts has an `@rpc` entrypoint.

The old `empty_config` wasn't really empty -- it had an `AMQP_URI` with an empty value, which was translated into `amqp://localhost`. The test failed if you ran it without having a broker running locally.